src/node_api_helpers.h

```diff
@@ -14,6 +14,7 @@
 #include "node_asyncapi.h"
 #include <limits.h>
 #include <string.h>
+#include <assert.h>
```

Comment:
It is, because code below this line makes assertions so, if you include this header without this modification, the resulting code fails to build.